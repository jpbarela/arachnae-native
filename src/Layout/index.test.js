// @flow
import * as React from "react";
import { render } from "@testing-library/react-native";
import { Row } from "./index";

test("Row renders", () => {
  const container = render(<Row />);

  expect(container).toMatchSnapshot();
});
