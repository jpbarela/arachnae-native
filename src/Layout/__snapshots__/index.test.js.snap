// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "alignItems": "flex-start",
        "justifyContent": "space-between",
      },
    ]
  }
/>
`;
